src/statmodels.jl

```diff
@@ -85,7 +85,7 @@ TestStat(x::TestStat) = x
 
 float(x::Union{TestStat, PValue}) = float(x.v)
 
-for op in [:(==), :<, :≤, :>, :≥, :(isless), :(isequal)] # isless and < to place nice with NaN
+for op in [:(==), :<, :≤, #==:>, :≥,==# :(isless), :(isequal)] # isless and < to place nice with NaN
```

Comment:
```suggestion
for op in [:(==), :<, :(isless), :(isequal)] # isless and < to place nice with NaN
```
Might as well remove rather than comment out. Also we don't need to define `<=` since having `==` and `<` suffices.

Reply:
`<=` is good to have as it has many optimized specializations. (e.g. for comparsion between floats and ints)